sdk/storage/azure-storage-blob-stress/src/main/java/com/azure/storage/blob/stress/BlobScenarioBase.java

```diff
@@ -85,7 +86,7 @@ public void run() {
             } else {
                 telemetryHelper.trackMismatch(span);
             }
-        } catch (Exception e) {
+        } catch (Throwable e) {
```

Comment:
Any reason for the change from `Exception` to `Throwable` while adding `Exception` as a checked exception to `RunInternal`?

Reply:
want to catch all `RuntimeExceptions` here and record them - we had some mystery around OOM going on here which was not caught/reported